docs/releasing.md

```diff
@@ -26,10 +26,10 @@ You must have docker installed.
 1. A container image of the shared cluster-api-provider-metal3 manager
 1. A git tag
 1. A release on Github containing:
-    - A manifest file - `infrastructure-components.yaml`
-    - A metadata file - `metadata.yaml`
-    - A cluster template - `cluster-template.yaml`
-    - A file containing an example of variables to set - `example_variables.rc`
+  - A manifest file - `infrastructure-components.yaml`
+  - A metadata file - `metadata.yaml`
+  - A cluster template - `cluster-template.yaml`
+  - A file containing an example of variables to set - `example_variables.rc`
```

Comment:
This does not look correct now as rendered by github. I think that extra space is needed.
![image](https://user-images.githubusercontent.com/9117693/205241813-1ce93d3c-9f90-4a13-80d4-1ecb8d3c5894.png)

Reply:
True, let me check, thanks for noticing.

Reply:
Done